lib/blueprinter/version.rb

```diff
@@ -1,3 +1,3 @@
 module Blueprinter
-  VERSION = '0.9.0'
+  VERSION = '0.9.1'
```

Comment:
Should this be a 0.10.0 since it's new feature?

Reply:
Thought simpler features could be accommodated in patch version updates itself.